src/View/Antlers/Language/Runtime/Sandbox/RuntimeValues.php

```diff
@@ -15,7 +15,7 @@ public static function resolveWithRuntimeIsolation($augmentable)
     {
         GlobalRuntimeState::$requiresRuntimeIsolation = true;
         try {
-            if ($augmentable instanceof Collection && $augmentable->count() && $augmentable[0] instanceof Augmentable) {
+            if ($augmentable instanceof Collection && $augmentable->count() && $augmentable->first() instanceof Augmentable) {
                 $value = BulkAugmentor::make($augmentable)->toArray();
```

Comment:
you actually don't need the `count` if you're doing `first`. It'll return null if there are none.